src/sentry/dynamic_sampling/rules/utils.py

```diff
@@ -185,3 +164,16 @@ def get_enabled_user_biases(user_set_biases: Optional[List[ActivatableBias]]) ->
 
 def get_supported_biases_ids() -> Set[str]:
     return {bias["id"] for bias in DEFAULT_BIASES}
+
+
+def eval_dynamic_factor(base_sample_rate: float, x: float) -> float:
```

Comment:
eval sounds a bit risky since it clash with python eval() function in my mind.
wdyt to rename it to `apply`?

Reply:
Sounds good!